### PR TITLE
JP-1423 Add exposure type NRS_VERIFY

### LINKED ISSUE #4878: Processing of NRS_VERIFY exposures
_Issue [JP-1423](https://jira.stsci.edu/browse/JP-1423) was created on JIRA by [James Muzerolle](https://jira.stsci.edu/secure/ViewProfile.jspa?name=muzerol):_

EXP_TYPE "NRS_VERIFY" was added last year to cover IFU and MOS verification images that may be taken by users to verify the status of target acquisition.  These exposures should be processed in the same way as the other NIRSpec imaging/target acquisition exposure types:
 * CALWEBB_DETECTOR1 (in full)
 * CALWEBB_IMAGE2 (partial)
 ** assign_wcs
 ** flat_field
 ** resample

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves [JP-1423](https://jira.stsci.edu/browse/JP-1423)
Resolves #4878 

Updates include:

- core schema to include NRS_VERIFY as a valid exposure type
- Generate Level2 associations consistent with other target acquisition/confirmation image types

### CODE REVIEW

File: jwst/datamodels/schemas/core.schema.yaml [hunk 1]
```diff
@@ -606,7 +606,8 @@ properties:
               # NIRSpec
               NRS_AUTOFLAT, NRS_AUTOWAVE, NRS_BRIGHTOBJ, NRS_CONFIRM, NRS_DARK,
               NRS_FIXEDSLIT, NRS_FOCUS, NRS_IFU, NRS_IMAGE, NRS_LAMP, NRS_MIMF,
-              NRS_MSASPEC, NRS_MSATA, NRS_TACONFIRM, NRS_TACQ, NRS_TASLIT, NRS_WATA,
+              NRS_MSASPEC, NRS_MSATA, NRS_TACONFIRM, NRS_TACQ, NRS_TASLIT,
+              NRS_VERIFY, NRS_WATA,
```
Comment (hbushouse) [inline jwst/datamodels/schemas/core.schema.yaml:610]: Thanks for maintaining the alphabetical listing!

**Review by hbushouse — APPROVED** [review]
LGTM

File: jwst/datamodels/schemas/core.schema.yaml (hunk 1 above)
Comment (stscieisenhamer) [reply jwst/datamodels/schemas/core.schema.yaml:610]: But of course! Is there any other way?

**Comment by nden** [comment]
This may need an entry in assign_wcs
https://github.com/spacetelescope/jwst/blob/master/jwst/assign_wcs/nirspec.py#L1750

**Pull request merged**